hermes-console-vue/src/api/hermes-client/index.ts

```diff
@@ -60,3 +61,17 @@ export function fetchOfflineClientsSource(
     `/topics/${topicName}/offline-clients-source`,
   );
 }
+
+export function queryTopics(queryJSON: object): ResponsePromise<Topic[]> {
+  return axios.post<Topic[]>(`/query/topics`, queryJSON, {
+    headers: { 'Content-Type': 'application/json' },
```

Comment:
I think we can extract this header and place it at the top of this file. Something like this:
```typescript
const contentTypeHeader = 'Content-Type'
const jsonMediaType = 'application/json'
```

Reply:
not relevant anymore